plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/Content.js

```diff
@@ -104,8 +106,10 @@ Ext.define('NX.controller.Content', {
     // install new feature view
     content.add(cmp);
 
-    // fire activate event to view component
-    cmp.fireEvent('activate', cmp);
```

Comment:
The purpose of that event is to make the new component visible, but removing it doesn’t seem to break anything. Are components that are added to containers automatically activated? @adreghiciu?